hc/settings.py

```diff
@@ -95,7 +95,7 @@
         'default': {
             'ENGINE':   'django.db.backends.postgresql',
             'NAME':     'hc',
-            'USER':     'postgres',
+            'USER':     'jameskatarikawe',
```

Comment:
We should maintain uniformity in the project files, lets keep user at postgres. line[98]